### PR TITLE
Review Page IN

### PR DESCRIPTION
**Pull request by jeewanbhusal**
### What this PR does (required):
- This PR contains integration of review frontend with the backend

### Screenshots / Videos (required):
Screen Shot 2021-08-04 at 16 55 57
Screen Shot 2021-08-04 at 16 54 25

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/components/ReviewTab/ReviewTab.tsx [hunk 1]
```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
             <ReviewTabForm handleSubmit={handleSubmit} />
           </Grid>
         </Grid>
-      </Grid>
+      </Paper>
+      <Typography align="center" component="h1" variant="subtitle1" className={classes.readAll}>
```
Comment (rajivtitus) [inline client/src/components/ReviewTab/ReviewTab.tsx]: Please reserve the usage of header tags for main titles/headings (for accessibility reasons)

File: client/src/components/ReviewTab/ReviewTab.tsx [hunk 2]
```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
...
+            <Box key={review._id} display="flex" justifyContent="center">
+              <Paper className={classes.paper}>
+                <Box display="flex" mb={2} textAlign="center" pt={2}>
+                  <Avatar alt="Profile Image" src={review.reviewerId.profilePic} className={classes.avatar}></Avatar>
+                  <Typography className={classes.username}>
+                    @{review.reviewerId.username}
+                  </Typography>
+                </Box>
+                <Box display="flex" mb={2} textAlign="center">
+                  <Rating value={review.rating} size="large" />
```
Comment (rajivtitus) [inline client/src/components/ReviewTab/ReviewTab.tsx]: Shouldn't review be a read only component here? :)

File: client/src/components/ReviewTab/ReviewTab.tsx [hunk 3]
```diff
@@ -1,35 +1,69 @@
...
+  useEffect(() => {
+    async function reviews() {
+      const reviews = await getReviews(artistId);
+      if (reviews !== null) {
+        setReviews(reviews);
+      } else {
+        new Error('Could Not Get Artist Info');
+      }
+    }
+    reviews();
```
Comment (rajivtitus) [inline client/src/components/ReviewTab/ReviewTab.tsx]: Please give this function a more descriptive name :)

File: client/src/components/ReviewTab/ReviewTab.tsx [hunk 4]
```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
...
           </Grid>
         </Grid>
-      </Grid>
+      </Paper>
+      <Typography align="center" component="h1" variant="subtitle1" className={classes.readAll}>
+        Read all
+      </Typography>
+      <Switch checked={checked} onChange={handleChange} inputProps={{ 'aria-label': 'controlled' }} />
+      {checked === true
+        ? reviews &&
```
Comment (rajivtitus) [inline client/src/components/ReviewTab/ReviewTab.tsx]: We may not need the short circuit operator as we are mapping over the reviews array here

**Review by rajivtitus — APPROVED** [review]
LGTM! Just some minor comments. Thanks

File: client/src/components/ReviewTab/ReviewTab.tsx (hunk 1 above)
Comment (jeewanbhusal-1) [reply client/src/components/ReviewTab/ReviewTab.tsx]: sure :)

File: client/src/components/ReviewTab/ReviewTab.tsx (hunk 3 above)
Comment (jeewanbhusal-1) [reply client/src/components/ReviewTab/ReviewTab.tsx]: fixing in the next commit :) thank you

File: client/src/components/ReviewTab/ReviewTab.tsx (hunk 4 above)
Comment (jeewanbhusal-1) [reply client/src/components/ReviewTab/ReviewTab.tsx]: handled that in the recent PR for artist page UI improvement

File: client/src/components/ReviewTab/ReviewTab.tsx (hunk 2 above)
Comment (jeewanbhusal-1) [reply client/src/components/ReviewTab/ReviewTab.tsx]: it should be :) ., changing it

**Pull request merged**